hardhat.config.js

```diff
@@ -60,6 +61,13 @@ module.exports = {
       gas: 7700000,
       gasPrice: 1 * GWEI,
     },
+    matic: {
```

Comment:
Again matic what? testnet or mainnet? which testnet? needs to be explicitly mentioned.

Reply:
+1